python/pipeline/meso.py

```diff
@@ -624,56 +607,59 @@ def _make_tuples(self, key):
         scan_filename = (experiment.Scan() & key).local_filenames_as_wildcard
         scan = scanreader.read_scan(scan_filename)
 
-        for field_id in range(scan.num_fields):
```

Comment:
Same as above. for loop for `scan.num_fields` is missing

Reply:
same answer